addons/mail/static/src/js/form_renderer.js

```diff
@@ -50,28 +77,21 @@ FormRenderer.include({
      * @private
      */
     _renderNode: function (node) {
-        var self = this;
         if (node.tag === 'div' && node.attrs.class === 'oe_chatter') {
-            if (!this.chatter) {
-                this.chatter = new Chatter(this, this.state, this.mailFields, {
-                    isEditable: this.activeActions.edit,
-                    viewType: 'form',
-                });
-
-                var $temporaryParentDiv = $('<div>');
-                this.defs.push(this.chatter.appendTo($temporaryParentDiv).then(function () {
-                    self.chatter.$el.unwrap();
-                    self._handleAttributes(self.chatter.$el, node);
-                }));
-                return $temporaryParentDiv;
-            } else {
-                this.chatter.update(this.state);
-                return this.chatter.$el;
-            }
+            return;
         } else {
             return this._super.apply(this, arguments);
         }
     },
+
+    /**
+     * @override
+     * @private
+     */
+    async _renderView() {
+        await this._super.apply(this, arguments);
+        await this._mount();
+    },
```

Comment:
not all form renderer should mount a chatter: only when there's a node `<div class="oe_chatter">`